integration/resources/expected/single/basic_api_with_mode_update.json

```diff
@@ -1,10 +1,10 @@
 [
         {"LogicalResourceId": "MyApi", "ResourceType": "AWS::ApiGateway::RestApi"},
-        {"LogicalResourceId": "MyApiDeploymentb30ecf9df1", "ResourceType": "AWS::ApiGateway::Deployment"},
+        {"LogicalResourceId": "MyApiDeployment", "ResourceType": "AWS::ApiGateway::Deployment"},
```

Comment:
Why are the hashes removed here?

Reply:
So the hashes are their to help guard against changes that accidentally change how SAM hashes those values. When the hashes change, this causes new resources in the stacks and can cause impact to. We had an event where we changed how Layers hashing was done and this caused throttling for customers.

Can we add these back?